src/entity/User.ts

```diff
@@ -0,0 +1,14 @@
+import { Entity, Column } from 'typeorm';
+import { BaseEntity } from './BaseEntity';
+
+@Entity('users')
+export class User extends BaseEntity {
+  @Column('text')
+  name: string;
+
+  @Column('text')
+  email: string;
```

Comment:
Please add the ﻿unique: true constraint to the email column. This will ensure that each email value is unique in the database